### DISCUSSION TITLE
USB Rubber Ducky components.

### DISCUSSION THREAD

**Discussion by WixOff16** [community/community#133972] [Programming Help]
### Select Topic Area

Question

### Body

Hey, 
I was wondering what would be the best way to create my own cheap USB rubber ducky, if I should make it with a raspberry pi pico W or with a digispark or?
What would be the best ?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @WixOff16 ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
